pkg/identity/keystone/authorizer.go

```diff
@@ -35,11 +35,6 @@ type Authorizer struct {
 }
 
 func resourceMatches(p policy, a authorizer.Attributes) bool {
-	if p.NonResourceSpec != nil && p.ResourceSpec != nil {
```

Comment:
The check is a duplicate of https://github.com/kubernetes/cloud-provider-openstack/blob/master/pkg/identity/keystone/authorizer.go#L154

Reply:
Ack